src/fixtures.ts

```diff
@@ -48,14 +80,17 @@ export async function withFixtures(
   const keep = !!process.env.INLINE_FIXTURES_KEEP;
   const dir = tmp.dirSync({ keep, unsafeCleanup: true });
 
-  await setupFixtures(dir.name, fixtures);
+  const inaccessibleFixtures = await setupFixtures(dir.name, fixtures);
 
   const origDir = process.cwd();
   process.chdir(dir.name);
 
   try {
     return await fn(dir.name);
   } finally {
+    inaccessibleFixtures.forEach((filePath: string) =>
+      fs.chmodSync(filePath, 0o777)
```

Comment:
Hi @pliski: I am not sure why this is necessary? If necessary, this probably should have a comment in the code why this is needed. 

(Thanks for your continued patiences on this. This is looking good to land once this is resolved).

Reply:
Hi @ofrobots thank you for your response and for your time! 
Unfortunately it is necessary to change back all files permissions otherwise tmp would not be allowed to delete the temp directory he has created.
I'm adding a comment.